### PR TITLE
:arrow_up: Upgrade @material packages to 0.19 versions

### PR DESCRIPTION
**Pull request by Kerrick**
- :boom: Change "accent" to "secondary" globally, as done by MDC-Web.
  https://github.com/material-components/material-components-web/pull/1116
- :sparkles: `{{mdc-button}}` now has `unelevated` boolean attribute (default: `false`)
- :sparkles: `{{mdc-button}}` now has `dense` boolean attribute (default: `false`)
- :sparkles: `{{mdc-button}}` now has `compact` boolean attribute (default: `false`)
- :sparkles: `{{mdc-textfield}}` now has `box` boolean attribute (default: `false`)

### CODE REVIEW

**Comment by Kerrick** [comment]
And of course, in the middle today's upgrade, [0.20 was shipped](https://github.com/material-components/material-components-web/blob/master/CHANGELOG.md#0200-2017-09-05).

File: tests/dummy/app/controllers/application.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import Ember from 'ember';
 export default Ember.Controller.extend({
   isDemoVisible: true,
   myProgress: 0.5,
+  isFirstSwitchOn: true,
```
Comment (vkkostin) [inline tests/dummy/app/controllers/application.js]: This seems unrelated and weird. It's weird because I remember doing this when I was working on the switch in one of my branches, but I also remember taking it out.

Comment (Kerrick) [reply tests/dummy/app/controllers/application.js]: It is unrelated, but it makes sure that the switch on the demo under "Active Switch" actually begins active.

Comment (vkkostin) [reply tests/dummy/app/controllers/application.js]: I see. 👌

File: addon/components/mdc-textfield.js [hunk 2]
```diff
@@ -212,12 +230,7 @@ export default Ember.Component.extend(MDCComponent, {
         get(component, 'inputKeydownHandlers').removeObject(handler);
       },
       getNativeInput() {
-        const  value = get(component, 'value');
-        return {
-          value,
-          disabled: get(component, 'disabled'),
-          checkValidity: () => get(component, 'valid')
-        };
+        return getElementProperty(component, 'querySelector', () => [null])('input, textarea')[0];
```
Comment (vkkostin) [inline addon/components/mdc-textfield.js]: Why do you need to pass in `() => [null]`. Just curious.

Comment (Kerrick) [reply addon/components/mdc-textfield.js]: Good catch - [`Element.querySelector()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelector) returns a single element, not a NodeList.

File: addon/components/mdc-textfield.js [hunk 3]
```diff
@@ -123,12 +135,18 @@ export default Ember.Component.extend(MDCComponent, {
    * @type {Object}
    */
   CLASS_NAMES: cssClasses,
+  rippleOptions() {
+    return {
+      isSurfaceActive: () => getElementProperty(this, 'querySelector', () => [{[MATCHES]: () => false}])('input, textarea')[0][MATCHES](':active')
```
Comment (KnownSubset) [inline addon/components/mdc-textfield.js]: Can we break out some of the anonymous functions to separate variables?  This line is a bit jam packed.
